### PR TITLE
Add networking advanced labs chapter

### PR DESCRIPTION
**Pull request by ManuStoessel**
I added a 15th chapter to the labs with the networking advanced topic

### CODE REVIEW

File: docker/15_networking-advanced [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+And have a look at the list of networks again with `docker network ls`.
+
+## Running containers on a network
+
+First let's start a container on our newly created network:
+
+```bash
+docker run -d --name search --net dev elasticsearch
```
Comment (stroebitzer) [inline docker/15_networking-advanced]: via nginx it works

**Comment by toschneck** [comment]
@ManuStoessel / @stroebitzer could we merge this one?

**Comment by ManuStoessel** [comment]
I cna't even remember opening this PR, so yes?

**Comment by toschneck** [comment]
@themue could your review it?

**Pull request merged**